test-dsl/src/main/groovy/com/smartbear/readyapi4j/dsl/assertions/AbstractAssertionsDelegate.groovy

```diff
@@ -0,0 +1,116 @@
+package com.smartbear.readyapi4j.dsl.assertions
+
+import com.smartbear.readyapi4j.assertions.AssertionBuilder
+import com.smartbear.readyapi4j.assertions.Assertions
+import com.smartbear.readyapi4j.assertions.ContainsAssertionBuilder
+import com.smartbear.readyapi4j.assertions.InvalidHttpStatusCodesAssertionBuilder
+import com.smartbear.readyapi4j.assertions.ValidHttpStatusCodesAssertionBuilder
+
+import static com.smartbear.readyapi4j.assertions.Assertions.contains
+import static com.smartbear.readyapi4j.assertions.Assertions.notContains
+
+/**
+ * An abstract class which contains common assertions applicable to multiple requests: SOAP, REST etc.
+ */
+abstract class AbstractAssertionsDelegate {
```

Comment:
I think the assertions that are HTTP request specific should be extracted into a separate class, maybe called HttpRequestAssertions etc, and then you can use a @Delegate assertion in the Soap- and REST delegate classes to make them accessible from the outside.

This could be done in a separate PR; it becomes more interesting once the JDBC requests are in I suppose.